src/ex4.py

```diff
@@ -0,0 +1,9 @@
+# Surprise me.
+
+def months(monthnum: int) -> str:
+    with31days : list[int] = [1,3,5,7,8,10,12]
+    if monthnum in with31days:
+        return "31 days"
+    if monthnum == 2:
+        return "28-29 days"
+    return "30 days"
```

Comment:
Va bene 👍

Però se vorresti ancora "migliorarlo" potresti eliminare i "magic number" e sostituirli con delle costanti o con degli enun.

Guarda "6. Don't use "magic numbers"" in https://testdriven.io/blog/clean-code-python/

Potresti sostituire == 2 con == FEBRUARY oppure potresti creare un enum "Month" e sostituire tutti i numeri con i nomi delle costanti